### DISCUSSION TITLE
Default endpoint hit log message

### DISCUSSION THREAD

**Discussion by t1nky** [RobinTail/express-zod-api#1716] [General]
Currently, when endpoint is hit, there is a log of "method: path" https://github.com/RobinTail/express-zod-api/blob/8649cf28536721e3dfc0e9c069543b1ec44fb1cb/src/routing.ts#L32

I use pino with pino-http and I get more rich log message, but I have no ability to disable the default message, as a result I get 2 lines in the log for the same operation.

Do you think this should not be in the core of the package? Maybe it should be in some default middleware that we can overwrite. Currently that default log takes about 1/5 of total number of lines for us (approx 1mil additional lines).

It is not a big deal, but more work doing filtering and adds up to some amount of useless logs. If there is any particular use of that line that I'm missing, please let me know! Thank you.

**Comment by RobinTail** [comment]
The idea behind this line is that you can disable it by setting the logging level higher than "info", to "warn" in particular.
The line has to be in the core because it tells what the core is doing before invoking your code — can be useful for debugging. If you find it redundant for your case (on production for example you would not need it), you set the logger level to "warn" and only track warnings and errors.
What I can also do is to decrease the severity of this line from "info" to "debug". So that you can keep track of infos and cut off the debugs.

Does anything of this work for you, @t1nky ?

**Comment by RobinTail** [comment]
I'm going to decrease those message severity to "debug" in v19, and I will add a possibility to set "info" as a minimal one in the built-in logger configuration, @t1nky 

#1748
